adapters/unicredit-adapter/src/main/java/de/adorsys/xs2a/adapter/service/impl/UnicreditAccountInformationService.java

```diff
@@ -13,13 +13,18 @@
 import de.adorsys.xs2a.adapter.service.impl.model.UnicreditStartScaProcessResponse;
 import de.adorsys.xs2a.adapter.service.model.*;
 
+import java.util.Arrays;
+import java.util.HashSet;
 import java.util.Map;
+import java.util.Set;
 
 import static de.adorsys.xs2a.adapter.http.ResponseHandlers.jsonResponseHandler;
 
 public class UnicreditAccountInformationService extends BaseAccountInformationService {
     private static final String AUTHENTICATION_CURRENT_NUMBER_QUERY_PARAM = "authenticationCurrentNumber";
     private static final String DEFAULT_PSU_IP_ADDRESS = "0.0.0.0";
+    private static final Set<String> POSSIBLE_PSU_ID_TYPE_VALUES = new HashSet<>(Arrays.asList("HVB_ONLINEBANKING", "UCEBANKINGGLOBAL"));
+    private static final String DEFAULT_PSU_ID_TYPE = "HVB_ONLINEBANKING";
```

Comment:
could you put this default to the possible list

Reply:
Done, thanks